src/utils.js

```diff
@@ -0,0 +1,94 @@
+import { wordDictionary2 } from './letters/WordDictionary2';
+import { wordDictionary3 } from './letters/WordDictionary3';
+import { wordDictionary4 } from './letters/WordDictionary4';
+import { wordDictionary5 } from './letters/WordDictionary5';
+import { letterPoints } from './letters/LetterPoints';
+
+  // Look in correct dictionary to see if word exists
+  function validWord(word) {
+    let wordDictionArray = [wordDictionary2, wordDictionary3, wordDictionary4, wordDictionary5];
+    let wordDictionary = wordDictionArray[word.length - 2];
+    return !!wordDictionary.find(item => {
```

Comment:
It seems like this is fast enough for you. But if you ever find it getting too slow you can always turn the word arrays into objects for easier lookup

Reply:
Ok